db/seeds.rb

```diff
@@ -15,3 +15,6 @@
 1.upto(4) do |i|
   kudo = Kudo.create(title: Faker::Beer.style, content: Faker::Coffee.notes, giver_id: "#{i}", receiver_id: "#{i+1}")
 end
+
+CompanyValue.create(title: "Patient" )
+CompanyValue.create(title: "Helpful" )
```

Comment:
small cosmetics - either add space before title, or remove space after second quotation mark

Reply:
I change it

Reply:
Actually with round brackets in general there should be no spaces - so it would like `CompanyValue.create(title: "Helpful") ` 
Space after the brackets is used with curly brackets - {} like: `hash = { foo: bar }`